enhancements/update/standalone-worker-surge.md

```diff
@@ -0,0 +1,295 @@
+resources) available for new pods to take the place of old pods from the node being drained. Consider the reductive example of 
+a static cluster with a single worker node. If there is an attempt to drain the node in this example, there is no additional worker 
+node capacity available to schedule new pods to replace the pods being drained. This can result in a stalled drain -- one that 
+does not terminate until there is external intervention.
+
+Stalled drains create a frustrating experience for operations teams as they require analysis and intervention. They can also
+make it impossible to predict when an update will complete -- complicating work schedules and communications. There are a number of reasons 
+drains can stall, but simple lack of spare worker node capacity is a common one. The reason is that running an over-provisioned cluster
+(i.e. with more worker nodes than actually required for the steady state workloads) is not always cost-effective. Running
+an extra node 24x7 simply to ensure capacity for the short period while another node is being drained is inefficient.
+
+One cost-effective approach to ensure capacity is called "surging". With a surge strategy, during an update, the platform
+is permitted to bring additional worker nodes online, to accommodate workloads being drained from existing nodes. After an 
+update concludes, the surged nodes are scaled down and the cluster resumes its steady state.
+
+HyperShfit Hosted Control Planes (HCP) already support the surge concept. HyperShift `NodePools`
+expose `maxUnavailable` and `maxSurge` as configurable options during updates: https://hypershift-docs.netlify.app/reference/api/#hypershift.openshift.io/v1beta1.RollingUpdate . 
+Unfortunately, standalone OpenShift, which uses `MachineConfigPools`, does not. To workaround this
+limitation for managed services customers, Service Delivery developed a custom Machine Update Operator (MUO)
+which can surge a standalone cluster during an update (see [reserved capacity feature](https://github.com/openshift/managed-upgrade-operator/blob/a56079fda6ab4088f350b05ed007896a4cabcd97/docs/faq.md)).
+
+### Preventing Other Stalled Drains
+
+As previously mentioned, there are other reasons that drains can stall. For example, `PodDisruptionBudgets` can
+be configured in such a way as to prevent pods from draining even if there is sufficient capacity for them
+to be rescheduled on other nodes. A powerful (though blunt) tool to prevent drain stalls is to limit the amount of time
+a drain operation is permitted to run before forcibly terminating pods and allowing an update to proceed. 
+`NodeDrainTimeout`, in HCP's `NodePools` allows users to configure this timeout.
+The Managed Update Operator also supports this feature with [`PDBForceDrainTimeout`](https://github.com/openshift/managed-upgrade-operator/blob/master/docs/faq.md).

```

Comment:
```suggestion
The Managed Upgrade Operator also supports this feature with [`PDBForceDrainTimeout`](https://github.com/openshift/managed-upgrade-operator/blob/master/docs/faq.md).

```

Reply:
lol - I'll get it right one of these times.